components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,135 @@
+- Compliance information about each node that shows the compliance state - This information includes passed and failed controls for each profile executed against that node
+
+A Data Feed is not a data tap.
+A Data Feed operates by doing the following:
+
+- Every 4 hours, the data-feed-service will aggregate the client runs and compliance reports from the previous 4 hours and send this information to the registered destinations. This time interval is 4 hours by default, but is configurable
+- If there are no destinations, aggregation will not occur
+- The data aggregates and sends in batches of 50 nodes at a time. The batch amount is 50 by default, but is configurable
+
+By default, only Admin users of Chef Automate may create and manage Data Feeds.
+
+### Adding a Data Feed Endpoint
+
+A single Data Feed instance connects to one endpoint.
+Create as many Data Feed instances as needed.
+
+To add a Data Feed endpoint in Chef Automate:
+
+![Setup Data Feed Page](/images/docs/filled_form_create_data_feed.png)
+
+1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
+2. Select **Create Data Feed**
+3. Enter a unique Data Feed name
+4. Enter the URL for your Data Feed End Point, including any specific port details
+5. Enter the Username and Password that your 3rd party endpoint requires for authentication
+6. Select **Test Data Feed** to begin validating the connection details
+7. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+
+### Edit a Data Feed Endpoint
+
+To edit a Data Feed endpoint in Chef Automate:
+
+1. In _Data Feeds_, select the Data Feed name to open its detail page
+1. Edit the Data Feed name or URL
+1. Use the **Save** button to save your changes
+
+### Delete a Data Feed Endpoint
+
+To delete a Data Feed endpoint in Chef Automate:
+
+1. In _Data Feeds_, select **Delete Data Feed** from the menu at the end of the table row
+1. Select **Delete Data Feed** to confirm permanent deletion of this Data Feed
+
+## Configuring Global Data Feed Behavior
+
+{{< info >}}
+The settings in `config.toml` apply across all configured Data Feed endpoints.
+{{< /info >}}
+
+Using the Chef Automate command-line tool, navigate to `/hab/svc/data-feed-service/config/config.toml`.
+Change one or more configuration parameters to reflect the desired global Data Feed behavior:
+
+- Update the `feed_interval` parameter to change the interval for the Data Feed collection. The default value is four hours
+- Update the `node_batch_size` parameter to change the number of sets of node data sent in each individual batch to your endpoint. The default value is 50 nodes
+- Use the `updated_nodes_only` setting to determine what data to include in each export. The default setting is `true`, which causes only recently changed data to aggregate. Set `updated_nodes_only` to `false` and all the latest data is sent for any nodes that have updated since the last export
```

Comment:
@davymcaleer What's the difference between the `true` and `false` setting in this description? Is there another parameter that determines what is considered "recently changed"?

Reply:
@mjingle - yeah this could maybe be expressed better....
- true = a node that has had changes to it only sends the data that has changed in reference to that node
- false = a node that has had changes to it sends all the data about that node, even the pieces that haven't changed